File: RikDanChatProj-main/SOAProjectDanielRiki/SOAProjectDanielRiki/soa-project-daniel-riki/src/app/models/gameModel.ts
import { Room } from "./room.model";

export class GameModel {
    userId: number =0; //מסמל מי לחץ על הכפתור איקס או עיגול
    whoStartedGame = 0;
    whoseTurn = 0;
    currentCell: number = 0;
    room = new Room();
}

File: RikDanChatProj-main/RikiDanGames/Users/GameModel.cs
namespace Users;

public class GameModel
{
	public int userId { get; set; }///מסמל מי לחץ על הכפתור איקס או עיגול
	public int whoStartedGame { get; set; } = 0;
	public int whoseTurn { get; set; } = 0;
	public int currentCell { get; set; }
	public Room room { get; set; }
}